### PR TITLE
Add primary action buttons and more actions menu

### LINKED ISSUE #5956: Deal with the overcrowded envelope actions menu
Hello mail friends, I think there are too many actions for the envelope component. The popover in the screenshot spans for half the height of my screen.

Screenshot 2022-01-17 at 14 22 39

2 possible solutions:
- Getting rid of some of those and having them only accessible within the content
- Having a "more actions" `ActionButton` that "routes" to a submenu within the popover like in Deck

https://user-images.githubusercontent.com/26852655/149777598-da31efad-e4aa-4d81-b39b-7342071b8e85.mov

cc @jancborchardt @nimishavijay

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
resolve #5956 

before
image

after:
image
image

### CODE REVIEW

**Review by nimishavijay — COMMENTED** [review]
Looks great design wise! Only a tiny detail: instead of `view-previous` could we use `chevron-left` for the left arrow icon in "More actions"? It is quite thin and seems to get lost with the background. Otherwise it looks great! :)

**Comment by JuliaKirschenheuter** [comment]
@nimishavijay @jancborchardt thank you both!
Icon is changed

**Comment by ChristophWurst** [comment]
@JuliaKirschenheuter could you please add before/after screenshots to your PR description for easier comparison? :)

**Comment by JuliaKirschenheuter** [comment]
Sure,
before
image

after:
image
image

**Review by nimishavijay — APPROVED** [review]

File: src/components/EnvelopePrimaryActions.vue [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<template>
+	<ul class="list-item-content__actions--primary">
+		<!-- @slot Provide the primary actions for the right side quick menu, inside of Actions -->
+		<slot
+			name="primaryActions" />
```
Comment (ChristophWurst) [inline src/components/EnvelopePrimaryActions.vue:5]: I'd suggest to replace the named slot by a default slot to simplify how the component is used

**Comment by GretaD** [comment]
not directly the scope of this pr, but the menu is also part of the draft mailbox and there are many actions that dont apply to draft

File: src/components/EnvelopePrimaryActions.vue (hunk 1 above)
Comment (ChristophWurst) [reply src/components/EnvelopePrimaryActions.vue:5]: @JuliaKirschenheuter :ping_pong: :)

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**